QEfficient/cloud/execute.py

```diff
@@ -38,7 +38,7 @@ def main(
 
     tokenizer = load_hf_tokenizer(model_name, cache_dir, hf_token)
 
-    batch_size = get_compilation_batch_size(qpc_path)
+    batch_size, ctx_len = get_compilation_batch_size(qpc_path)
```

Comment:
Since we're returning multiple results and not just batch_size, we should rename this to something generic like `get_compilation_dims`

Reply:
Agree.